spec/ics-4-channel-and-packet-semantics/dataflow.tex

```diff
@@ -0,0 +1,21 @@
+\documentclass[tikz,10pt,border=10pt]{standalone}
+\usepackage{textcomp}
+\usepackage{xcolor}
+\usetikzlibrary{shapes,arrows}
+\begin{document}
+\tikzset{%
+  state/.style    = {draw, thick, rectangle, node distance = 6cm}
+}
+\newcommand{\CA}[1]{\it{A} \textcolor{black}{\textsc{#1}}}
+\newcommand{\CB}[1]{\it{B} \textcolor{black}{\textsc{#1}}}
+\newcommand{\CS}[2]{\begin{tabular}{l}
+\CA{#1} \\
+\CB{#2}
+\end{tabular}}
+
+\begin{tikzpicture}[auto, thick, node distance=2cm, >=triangle 45]
+
+\draw (0,0) -- (4,0) -- (4,4) -- (0,4) -- (0,0);
```

Comment:
This is incomplete, right? (It will basically just draw a rectangle as far as I understand)

Reply:
Yes, working on it!

Reply:
(but it is a pretty rectangle)

Reply:
Now features actual dataflow :)